freeipa-api/src/main/java/com/sequenceiq/freeipa/api/v1/freeipa/user/UsersyncV1Endpoint.java

```diff
@@ -15,19 +19,25 @@
 import io.swagger.annotations.Api;
 import io.swagger.annotations.ApiOperation;
 
-@Path("/v1/freeipa/usersync")
+@Path("/v1/freeipa/user")
 @Consumes(MediaType.APPLICATION_JSON)
-@Api(value = "/v1/freeipa/usersync", description = "Synchronize users to FreeIPA", protocols = "http,https")
+@Api(value = "/v1/freeipa/user", description = "Synchronize users to FreeIPA", protocols = "http,https")
 public interface UsersyncV1Endpoint {
```

Comment:
shall we change the interface name to follow the change in the path? I mean to UserV1Endpoint

Reply:
done